src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// Class representing a User
+public class User {
+  private String id;
+  private String email;
+  private String username;
+  private ArrayList<Long> planner;
+  private ArrayList<Long> cookbook;
+  private ArrayList<Long> userRecipes;
+  // shopping list will not be implented until after MVP
+  private ArrayList<String> shoppingList;
+
+  public User(String id, String email, String username) {
+    this.id = id;
+    this.email = email;
+    this.username = username;
+    planner = new ArrayList<Long>();
+    cookbook = new ArrayList<Long>();
+    userRecipes = new ArrayList<Long>();
+  }
+
+  public User(String id, String email, String username, ArrayList<Long> planner, ArrayList<Long> cookbook, ArrayList<Long> userRecipes) {
+    this.id = id;
+    this.email = email;
+    this.username = username;
+    this.cookbook = cookbook;
+    this.planner = planner;
+    this.userRecipes = userRecipes;
+  }
+
+  public void setID(String id) {
+    this.id = id;
+  }
+
+  public boolean equalIDs(String id) {
+    if (this.id.equals(id)) {
+      return true;
+    }
+    return false;
```

Comment:
you can just do: `return this.id.equals(id)`

Alternatively, since this operation is super simple, maybe it would make more sense to have a getID method and then wherever you're calling this from can check if the IDs are equal

Reply:
it feels sketchy to return the user ID though
shouldn't that be private? or does it not really matter?

Reply:
Oh that's an interesting point. I guess it depends on whether you think the user ID is sensitive, which in my opinion depends on where/how you're accessing it and what is the potential danger of it being exposed (although I'm not a security expert). I think it's fine to leave this as an equals method in that case.